podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java

```diff
@@ -284,6 +322,74 @@
     }
 
     /**
+     * GET /requests/validate/:uuid : Validate a request with uuid.
+     *
+     * @param uuid the uuid of the request to validate
+     * @return the ResponseEntity with the validated request representation
+     *
+     * @throws ActionNotAllowedInStatus when a requested action is not available for the status of the Request.
+     */
+    @GetMapping("/requests/validate/{uuid}")
```

Comment:
I'm in favour of using `/requests/{uuid}` to refer to a request consistently and appending actions after that.